docarray/array/mixins/io/json.py

```diff
@@ -53,21 +55,21 @@ def load_json(
             constructor = Document.from_dict
 
         with file_ctx as fp:
-            return cls(constructor(v, protocol=protocol, **kwargs) for v in fp)
+            return cls(constructor(v, protocol=protocol) for v in fp)
 
     @classmethod
     def from_json(
-        cls: Type['T'], file: Union[str, TextIO], protocol: str = 'jsonschema', **kwargs
+        cls: Type['T'],
+        file: Union[str, TextIO],
+        protocol: str = 'jsonschema',
     ) -> 'T':
-        return cls.load_json(file, protocol=protocol, **kwargs)
```

Comment:
if this is targeting master, then this diff shows you over-delete something which supposed to be there

Reply:
good catch, reverted